easybuild/easyconfigs/f/freetds/freetds-1.2-GCCcore-9.3.0.eb

```
@@ -0,0 +1,37 @@
+# easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org
+
+easyblock = 'ConfigureMake'
+
+name = 'freetds'
+version = '1.2'
+
+homepage = "https://www.freedesktop.org/"
+description = """FreeTDS is a set of libraries for Unix and Linux that allows your programs to
+ natively talk to Microsoft SQL Server and Sybase databases."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['ftp://ftp.freetds.org/pub/freetds/stable']
+sources = [SOURCE_TAR_GZ]
+checksums = ['c0a7c8bacd0a3afbc19cffa096b68f28d7470efcb6797ec51a4fa369cadd79a4']
+
+builddependencies = [('binutils', '2.34')]
+
```

It needs `libreadline` and also `OpenSSL`. For `OpenSSL` we prefer the system library, but we can add a comment to avoid the dependency go unnoticed.
```suggestion

dependencies = [
    ('libreadline', '8.0'),
    # ('OpenSSL', '1.1.1e'),  # for security reasons, OS dependency should be preferred if it is more recent
]

```